mesos-simulation/src/main/scala/mesosphere/mesos/simulation/DriverActor.scala

```diff
@@ -45,9 +48,21 @@ class DriverActor(schedulerProps: Props) extends Actor {
 
   private[this] val numberOfOffersPerCycle: Int = 10
 
+  private val randomSeed: Long = 123
+  // use seed for getting more deterministic behavior
+  private[this] val random = new Random(new java.util.Random(randomSeed))
```

Comment:
What's the reasoning behind this? Wouldn't be potentially cover more cases in the long run if we don't use a constant seed?